lib/ansible/modules/windows/win_firewall_rule.ps1

```diff
@@ -291,7 +282,12 @@ $fwsettings.Add("LocalPort", $localport);
 $fwsettings.Add("RemotePort", $remoteport);
 $fwsettings.Add("Service", $service);
 $fwsettings.Add("Protocol", $protocol);
-$fwsettings.Add("Profiles", $winprofile)
+
+If ($profiles -eq "any") {
+    $fwsettings.Add("Profiles", "Domain,Private,Public")
+} Else {
+    $fwsettings.Add("Profiles", (Get-Culture).textinfo.totitlecase($profiles))
```

Comment:
It seems it can be removed. Comparison operator `-eq` are [case-insensitive by default](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.core/about/about_comparison_operators):
>By default, all comparison operators are case-insensitive. To make a comparison operator case-sensitive, precede the operator name with a `c`. For example, the case-sensitive version of `-eq` is `-ceq`. To make the case-insensitivity explicit, precede the operator with an `i`. For example, the explicitly case-insensitive version of `-eq` is `-ieq`.